macros/edr/data_monitoring/monitors_query/column_monitoring_query.sql

```diff
@@ -33,7 +33,7 @@
         filtered_monitored_table as (
             select {{ column_obj.quoted }},
                    {%- if dimensions -%} {{ elementary.select_dimensions_columns(dimensions, "dimension") }}, {%- endif -%}
-                   {{ elementary.null_timestamp() }} as start_bucket_in_data,
+                   {{ elementary.null_timestamp() }} as start_bucket_in_data
```

Comment:
Was this flow broken? 🤔

Reply:
Yes, `column_anomalies` without `timestamp_column` was broken.
Added [this test](https://github.com/elementary-data/dbt-data-reliability/pull/728/commits/dd437b854464bdc489539409d5b9b5d00f91821f).